labrad/devices.py

```diff
@@ -327,12 +327,14 @@ def refresh_devices(self, c):
         returnValue(self.list_devices(c))
 
     @setting(1000001, 'Lock Device',
-                      data=[': Lock the selected device',
+                      data=[': Lock the selected device for default time',
                             'v[s]: Lock for specified time'],
                       returns=[''])
     def lock_device(self, c, data):
         """Lock a device to be accessible only in this context."""
         dev = self.selectedDevice(c)
+        if data:
```

Comment:
I'd suggest renaming this parameter to `timeout`. Also, the check here should be against None:
```python
if timeout is not None:
    ...
```
because this should also apply if `timeout == 0*ns`